ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1,1591 +1,535 @@
+		E7A30F04BECE3BF900000000 /* RCTFBSDKAccessToken.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29BECE3BF900000000 /* RCTFBSDKAccessToken.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04C57E516600000000 /* RCTFBSDKGraphRequestConnectionContainer.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29C57E516600000000 /* RCTFBSDKGraphRequestConnectionContainer.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04D84F937100000000 /* RCTFBSDKShareHelper.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29D84F937100000000 /* RCTFBSDKShareHelper.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04EBCCFB3700000000 /* RCTFBSDKSendButtonManager.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29EBCCFB3700000000 /* RCTFBSDKSendButtonManager.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04ED6CD2D500000000 /* RCTFBSDKShareAPI.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29ED6CD2D500000000 /* RCTFBSDKShareAPI.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04F12CB92500000000 /* RCTConvert+FBSDKLogin.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29F12CB92500000000 /* RCTConvert+FBSDKLogin.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04F7DF836100000000 /* RCTFBSDKLikeControlManager.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29F7DF836100000000 /* RCTFBSDKLikeControlManager.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+		E7A30F04F96A828B00000000 /* RCTFBSDKShareDialog.m in Sources */ = {isa = PBXBuildFile; fileRef = 1DD70E29F96A828B00000000 /* RCTFBSDKShareDialog.m */; settings = {COMPILER_FLAGS = "-stdlib=libc++ -D_LIBCPP_HAS_NO_STRONG_ENUMS=1"; }; };
+/* End PBXBuildFile section */
+
+/* Begin PBXFileReference section */
+		1DD70E29001F47FB00000000 /* BUCK */ = {isa = PBXFileReference; explicitFileType = text.script.python; path = BUCK; sourceTree = SOURCE_ROOT; };
+		1DD70E29001F47FB00000001 /* BUCK */ = {isa = PBXFileReference; explicitFileType = text.script.python; path = BUCK; sourceTree = SOURCE_ROOT; };
+		1DD70E29001F47FB00000002 /* BUCK */ = {isa = PBXFileReference; explicitFileType = text.script.python; path = BUCK; sourceTree = SOURCE_ROOT; };
+		1DD70E29001F47FB00000003 /* BUCK */ = {isa = PBXFileReference; explicitFileType = text.script.python; path = BUCK; sourceTree = SOURCE_ROOT; };
+		1DD70E2900B6388300000000 /* libRNSDKShareKit.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; name = libRNSDKShareKit.a; path = lib.a; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
`path = lib.a` on my machine means XCode will compile multiple different libraries into `lib.a` causing all sorts of troubles.
I have to manually set Product Name on each of the targets to stop that from happening.
Otherwise this is a change I'd really love to see merged into master.